doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -0,0 +1,449 @@
+.. _bps-preqs:
+
+Prerequisites
+-------------
+
+#. LSST software stack.
+
+#. Shared filesystem for data.
+
+#. Shared database.
+
+   `SQLite3`__ is fine for small runs like **ci_hsc_gen3** if have POSIX
+   filesystem.  For larger runs, use `PostgreSQL`__.
+
+#. HTCondor's `Python bindings`__.
```

Comment:
Don't think the htcondor python bindings are required if using Pegasus.  And if using Pegasus, need Pegasus software installed.